File: backend-crud-api/src/main/kotlin/com/jython/todo/api/task/application/port/in/CreateTaskUseCase.java
package com.jython.todo.api.task.application.port.in;

import com.jython.todo.api.task.application.port.in.command.CreateTaskCommand;
import com.jython.todo.api.task.domain.Task;

public interface CreateTaskUseCase {

  Task create(CreateTaskCommand command);
}

File: backend-crud-api/src/main/kotlin/com/jython/todo/api/task/application/port/in/CreateTaskUseCase.kt
package com.jython.todo.api.task.application.port.`in`

import com.jython.todo.api.task.application.port.`in`.command.CreateTaskCommand
import com.jython.todo.api.task.domain.Task

interface CreateTaskUseCase {
    fun create(command: CreateTaskCommand): Task
}